### PR TITLE
Detail context configuration and add timestamp documentation.

### PR DESCRIPTION
**Pull request by jonsnyder**
## Additional Details

I changed the link to the date format because if you look up the "date-time" datatype for json schemas, that link if where you finally end up.

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: reference/automatic-information.md [hunk 1]
```diff
@@ -101,3 +101,12 @@ The URL of the current page
 **Example:** `http://somereferrer.com/linkedpage.html`  
 
 The URL of the previous page visited
+
+## Timestamp
+
```
Comment (dancingcactus) [inline reference/automatic-information.md]: Is it possible to set your own timestamp?

File: reference/automatic-information.md [hunk 2]
```diff
@@ -101,3 +101,12 @@ The URL of the current page
...
+The timestamp of the event.  This part of context cannot be removed.
```
Comment (dancingcactus) [inline reference/automatic-information.md]: timestamps? Is this different than the local time above? Which one will take precedence.

Comment (jonsnyder) [reply reference/automatic-information.md]: They are the same timestamp, but local time is in the local timezone.

File: reference/automatic-information.md (hunk 1 above)
Comment (jonsnyder) [reply reference/automatic-information.md]: no

File: fundamentals/configuring-the-sdk.md [hunk 3]
```diff
@@ -58,6 +58,13 @@ Indicates whether debugging messages should be displayed in the browser's JavaSc
 
 Indicates whether errors should be suppressed. As described in [Executing Commands](executing-commands.md), _uncaught_ errors will be logged to the developer console, regardless of whether logging is enabled in Alloy. By setting `errorsEnabled` to `false`, promises returned from Alloy will never be rejected, though errors will still be logged to the console if logging is enabled in Alloy. 
 
+### `context`
+ **Type:** Array of Strings\
+ **Required:** No\
+ **Default Value:** `["web", "device", "environment", "placeContext"]`
+
+Indicates which context catagories to collect automatically as described in [Automatic Information](reference/automatic-information.md).  If this configuration is not specified all of the categories will be used by default.
```
Comment (Aaronius) [inline fundamentals/configuring-the-sdk.md]: "specified all of the categories" -> "specified, all categories"

File: reference/automatic-information.md [hunk 4]
```diff
@@ -7,9 +7,9 @@ description: >-
 This documentation is for a library and a service that is in Alpha and should not be used for production use-cases.
 {% endhint %}
 
-The Adobe Experience Cloud SDK collects a number of pieces of information automatically without any special configuration, however, they can be disabled if needed using the `configure` command [See Configuring the SDK](fundamentals/configuring-the-sdk.md). Below is a list of those pieces of information.
+The Adobe Experience Cloud SDK collects a number of pieces of information automatically without any special configuration, however, they can be disabled if needed using the `context` option in the `configure` command [See Configuring the SDK](fundamentals/configuring-the-sdk.md). Below is a list of those pieces of information. (The name in parentheses indicates the string to use when configuring the context)
```
Comment (Aaronius) [inline reference/automatic-information.md]: I'd remove the parenthesis and put a period at the end.

**Pull request merged**